.travis.yml

```diff
@@ -100,5 +100,18 @@ gemfile:
   - gemfiles/rails_4.2.5.1.gemfile
   - gemfiles/rails_4.2.5.2.gemfile
   - gemfiles/rails_4.2.6.gemfile
+  - gemfiles/rails_4.2.7.gemfile
+  - gemfiles/rails_4.2.7.1.gemfile
+
+matrix:
+  include:
+    - rvm: 2.2.2
+      gemfile: gemfiles/rails_5.0.1.gemfile
+      gemfile: gemfiles/rails_5.0.0.gemfile
+      gemfile: gemfiles/rails_5.0.0.1.gemfile
+    - rvm: 2.3.1
```

Comment:
@jonkessler as @n-rodriguez wrote, we need more recent versions of Ruby (2.2.2+) to test with Rails 5. Let us know if you have a better solution for how to configure this in travis.yml. This is just the most straightforward approach, we test against the most recent from 2.2 and 2.3 branches.